### DISCUSSION TITLE
Welcome to smart-money-concepts Discussions!

### DISCUSSION THREAD

**Discussion by joshyattridge** [joshyattridge/smart-money-concepts#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satheesh1503** [comment]
I am really thankful for your attempt to create an indicator for SMC logics. As I am new to GitHub, I am not sure how to install this indicator on the MT4 platform to test it. Despite my 15+ years of experience in trading, this aspect is new to me. Could you kindly provide me with the process for connecting the indicator with MT4?

Additionally, I am interested in knowing if it is possible to display all higher time frame liquidity, order blocks (OB), and fair value gaps (FVG) in the same window. Also, do you use any alerts for when the price touches the Points of Interest (POI)?

Thank you for your assistance!

**Reply by joshyattridge** [reply]
Hello @satheesh1503,

This is a python package there for it isn't build for MT4, this support is something you would need to implement yourself if you need it.

Regards,
Josh

**Reply by nessos666** [reply]
Hi Josh,

  I came across your smart-money-concepts library on GitHub and I'm really impressed by
  your work — especially the swing deduplication, the 4-point BOS/CHoCH pattern, and the
  FVG mitigation tracking. Very clean and well thought out.

  I'm working on a similar project — an algorithmic NQ/MNQ futures trading system built in
  Python. I've implemented ICT concepts like Order Blocks, BOS, FVG, Breaker Blocks,
  Liquidity Sweeps, and more. I've been running 13+ strategies in parallel with a full
  infrastructure (Telegram monitoring, safety layer, signal combiner, backtesting engine).

  I'd really love to connect and exchange ideas. I think we could both benefit from each
  other's perspectives — I'm happy to share everything I've built, no strings attached. I
  just want to connect with like-minded people who are serious about algo trading.

  I'm not very active on big social media platforms (bit of a nerd 😄), but I'm easy to
  reach on:
  - Telegram
  - Signal
  - WhatsApp
  - Email

  Unfortunately I don't speak English well enough for voice calls, but written
  communication works great (I use translation tools). I'm based in Germany.

  Would be awesome to hear from you!

  Best regards,
  David
mikodavid(at)web.de
